Makefile

```diff
@@ -1,21 +1,25 @@
 APP_VERSION ?= $(shell git describe --abbrev=5 --dirty --tags --always)
 BINDIR ?= $(PWD)/bin
-CHARTDIR ?= $(PWD)/chart/stack
+CHARTDIR ?= $(PWD)/charts
 
 OS ?= $(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH ?= amd64
 
 PATH := $(BINDIR):$(PATH)
 SHELL := env 'PATH=$(PATH)' /bin/sh
 
-.PHONY: chart
-chart:
-	yq w -i $(CHARTDIR)/Chart.yaml version "$(APP_VERSION)"
-	yq w -i $(CHARTDIR)/Chart.yaml appVersion "$(APP_VERSION)"
+.PHONY: charts
+charts: chart-stack chart-wordpress-site
 	yq w -i $(CHARTDIR)/values.yaml nginx-ingress.defaultBackend.image.tag "$(APP_VERSION)"
 
+chart-%:
+	yq w -i $(CHARTDIR)/$*/Chart.yaml version "$(APP_VERSION)"
```

Comment:
it's better, for now, to have it under the same tag `charts`
Because when the first tag is executed will modify files and when APP_VERSION (`git describe...`) will be executed again will return a tag with `-dirty`. See last drone logs.